src/assets/index.js

```diff
@@ -0,0 +1,18 @@
+import categoryBeauty from './fashion-photography-of-woman-hands-on-chin-with-glitter-1081685.jpg';
+import categoryRestaurants from './delicious-vegetable-stew-with-fish-in-black-ceramic-bowl-4194617.jpg';
+import categoryServices from './boy-sitting-trimmed-by-man-holding-scissor-2090771.jpg';
+import categoryNonprofits from './group-of-people-standing-on-street-3607345.jpg';
+import categoryRetail from './photo-of-black-clothes-on-hangers-1036856.jpg';
+import contactBanner from './maryellens.jpg';
+import discoveryPageTitleBanner from './Discovery Title Graphic.svg';
+
+// TODO: Source images from remote source potentially maybe
```

Comment:
Should this be a `nice to have` issue?

Reply:
Potentially? We can talk about it during a standup. In any case, I like to take quick notes of fleeting thoughts while I develop and this is one of them. Not a pressing issue since it's nowhere near necessary until we get to multi-city support.